fuzzit.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -ex
```

Comment:
I think it's necessary to check the command and install them if not exist, such as `go-fuzz-build`, `clang-9` and so on. @yevgenypats